R/count_teleconnections.R

```diff
@@ -289,11 +290,16 @@ count_watershed_teleconnections <- function(data_dir,
           x$metrics %>% filter(metric == "cultivated runoff") %>%
             .[["value"]] -> cropland_runoff
 
-          tibble(total_runoff, dev_runoff, cropland_runoff) %>%
-            mutate(nonpristine_conc = (dev_runoff + cropland_runoff) / total_runoff,
-                   ag_conc = cropland_runoff / total_runoff,
-                   dev_conc = dev_runoff / total_runoff) %>%
-            select(nonpristine_conc, ag_conc, dev_conc)
+          as.numeric(total_runoff) -> total_runoff_num
+          as.numeric(dev_runoff) -> dev_runoff_num
+          as.numeric(cropland_runoff) -> cropland_runoff_num
```

Comment:
I'm gonna revert this back to what it was and we can review the error you had on your machine. There shouldn't be any need to convert to numeric as these variables are already stored as numerical variables.